src/main/java/org/ibp/api/java/impl/middleware/germplasm/GermplasmListServiceImpl.java

```diff
@@ -754,6 +748,18 @@ private Integer getFolderIdAsInteger(final String folderId) {
 		return (CROP_LISTS.equals(folderId) || PROGRAM_LISTS.equals(folderId)) ? null : Integer.valueOf(folderId);
 	}
 
+	private static GermplasmListDto transformGermplasmList(GermplasmList input) {
```

Comment:
@lik-83 Much better to have a mapper to do this transform

Reply:
Done